test/c++/Makefile

```diff
@@ -0,0 +1,24 @@
+include ../common.mak
+
+TESTS:=exceptions
+
+ifeq (osx,$(OS))
+	LDSTDCPP := -L-lc++
+else
+	LDSTDCPP := -L-lstdc++
+endif
+
+.PHONY: all clean exceptions
+all: $(addprefix $(ROOT)/,$(addsuffix .done,$(TESTS)))
+clean:
+	rm -rf $(GENERATED)
+
+# Generic testing rule
+$(ROOT)/%.done: $(ROOT)/%
+	@echo Testing $*
+	$(QUIET)$(TIMELIMIT)$(ROOT)/$* $(RUN_ARGS)
+	@touch $@
+
+$(ROOT)/exceptions:
+	$(QUIET)$(CXX) exceptions/exception.cpp -c -o exceptions/cpp_exception_test.o
```

Comment:
How do you manipulate this makefile to build with MSVC?

Reply:
Ergh... I don't think this testsuite works on Windows at all.

Reply:
The auto tester runs on most platforms that DMD targets... how do those tests run?

Reply:
https://auto-tester.puremagic.com/pull-history.ghtml?projectid=1&repoid=2&pullid=2303

The tests pass on Windows

Reply:
yeah, i spotted that... I have no idea how; that's not even remotely a valid compile command line :/

Reply:
When I said "works on Windows", I meant "is run". That's why the test pass, because it just does not run this.